webpack.config.js

```diff
@@ -139,18 +148,24 @@ module.exports = function makeWebpackConfig () {
    * Reference: https://github.com/postcss/autoprefixer-core
    * Add vendor prefixes to your css
    */
-  config.postcss = [
-    autoprefixer({
-      browsers: ['last 2 version']
-    })
-  ];
+   // NOTE: This is now handled in the `postcss.config.js`
+   //       webpack2 has some issues, making the config file necessary
```

Comment:
@Foxandxss I believe this is where i got my information (near bottom of thread, stating `postcss.config.js` is required when using postcss plugins)

https://github.com/postcss/postcss-loader/issues/125

![image](https://cloud.githubusercontent.com/assets/183153/22273076/354943b6-e253-11e6-892f-b6dd28cec52c.png)